prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -2029,6 +2034,11 @@ subroutine initProgramVariables(this, input, env)
       else if (allocated(input%ctrl%dispInp%mbd)) then
         if (this%isLinResp) then
           call error("MBD model not currently supported for Casida linear response")
+        else if (allocated(this%reks)) then
+          call error("Selfconsistent MBD/TS dispersion is blocked from REKS")
+          if (this%tForces) then
+            call error("Calculation of selfconsistent MBD/TS is not compatible with force calculation in REKS")
```

Comment:
```suggestion
            call error("Calculation of self-consistent MBD/TS is not currently compatible with force calculation in REKS")
```

Reply:
I changed the sentence based on your suggestions.

Reply:
This is a bit confusing, because we currently don't have a self-consistent MBD/TS implementation.

Reply:
Then, which sentence do you prefer? How about "MBD/TS model is not currently compatible with force calculation in REKS"?